index.html

```diff
@@ -36,7 +36,217 @@ <h2 class="sd">
                 </ul>
                 <span class="dis"> 
                     <img src="./images/Disabled.png" alt="disabled">
-                </span>
-               
+                </span>    
+                </section>
+
+                <section class="card-sections">
+                    <h1><br>My Recent <br>Works</h1>
+                    
+                    <div class="indicator">
+                    <a href="#"><img src="./images/Indicator.png" alt="#"></a>
+                </div>
+                    <div class="container-card">
+                    <div class="placeholder">         
+                    </div>
+                    <div class="card-lists">
+                        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+                        <div class="items">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>Css</li>
+                            <li>Java script</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                    <div class="button-cont">
+                        <input type="button" value="see project" class="potfolio-but">
+                    </div>
```

Comment:
- [ ] Please the button "See project" is a link `<a>` or `<button>` tag
- [ ]  The button styling: Active, Hover, and Pressed (see design)
Example:
![hover](https://user-images.githubusercontent.com/63972670/160213942-a8ad1082-cda4-426b-b4d9-70b2f8aceed9.JPG)